test-utils/build.gradle

```diff
@@ -24,17 +24,24 @@ dependencies {
         exclude group: 'org.slf4j', module: 'slf4j-reload4j'
     }
 
+
+    // The following dependencies needs to be excluded because they do not have licenses which we are allowed to include
+    def withoutLicenseViolations = {
+        exclude group: 'com.sun.codemodel'
```

Comment:
This has the license: https://github.com/javaee/jaxb-codemodel/tree/master, the repo is archived which may be why the license tool has issues with it, you can manually add it in to the licenses instead. See line 117 of licenses-3rdparties.gradle on how to do this.